### PR TITLE
Add login and register pages with working DB

### PR DESCRIPTION
**Pull request by medikornov**

### CODE REVIEW

File: Data/FoodContext.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+namespace FoodKept.Data
+{
+    public class FoodContext : IdentityDbContext
+    {
+        public FoodContext (DbContextOptions<FoodContext> options) : base(options)
+        {
+
+        }
+        //public DbSet<User> Users { get; set; }
```
Comment (ArmantasPik) [inline Data/FoodContext.cs:20]: Comment could have been deleted.

File: Pages/Register.cshtml.cs [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+        public RegisterModel (SignInManager<IdentityUser> signInManager, UserManager<IdentityUser> userManager)
+        {
+            _signInManager = signInManager;
+            _userManager = userManager;
+        }
+
+
+        [BindProperty]
+        public User login { get; set; }
```
Comment (ArmantasPik) [inline Pages/Register.cshtml.cs:28]: For styling, maybe this could be next to other properties :D

File: Pages/Login.cshtml.cs [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+        private readonly SignInManager<IdentityUser> _signInManager; 
+
+        public LoginModel(SignInManager<IdentityUser> signInManager)
+        {
+            _signInManager = signInManager;
+        }
+
+        [BindProperty]
+        public Login Model { get; set; }
```
Comment (ArmantasPik) [inline Pages/Login.cshtml.cs:26]: Could be next to other property

File: Pages/Login.cshtml.cs [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+        public void OnGet()
+        {
+        }
+        public async Task<IActionResult> OnPostAsync(string returnUrl = null)
+        {
+            if(ModelState.IsValid)
+            {
+                var result = await _signInManager.PasswordSignInAsync(Model.Username, Model.Password, false, false);
```
Comment (ArmantasPik) [inline Pages/Login.cshtml.cs:34]: Add a remember me?

File: Pages/Login.cshtml.cs [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+                    }
+                }
+
+                ModelState.AddModelError("", "Username or password is incorrect");
+            }
+            return Page();
+        }
+    }
+
+    public class Login
```
Comment (ArmantasPik) [inline Pages/Login.cshtml.cs:53]: Could be in another file

**Review by ArmantasPik — APPROVED** [review]
Review for PR

File: Program.cs [hunk 6]
```diff
@@ -22,5 +46,7 @@ public static void Main(string[] args)
                 {
                     webBuilder.UseStartup<Startup>();
                 });
+
```
Comment (0rl1s) [inline Program.cs:49]: enter enter :D

**Pull request merged**